nipype/interfaces/afni/preprocess.py

```diff
@@ -2789,6 +2790,13 @@ class WarpInputSpec(AFNICommandInputSpec):
         argstr='-zpad %d')
     verbose = traits.Bool(
         desc='Print out some information along the way.', argstr='-verb')
+    save_warp = traits.Bool(
+        False, usedefault=True, desc='save warp as .mat file')
+
+
+class WarpOutputSpec(TraitedSpec):
+    out_file = File(desc='Warped file.', exists=True)
+    warp_file = File(desc='warp transfrom .mat file')
```

Comment:
s/transfrom/transform/

Reply:
what do you mean ?

Reply:
Oh, sorry. That's regular expression. I just meant you misspelled "transform".

Reply:
Sorry ! I am correcting that